cvat/apps/annotation/yolo.py

```diff
@@ -0,0 +1,129 @@
+format_spec = {
+    "name": "YOLO",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "ZIP",
+            "version": "1.0",
+            "handler": "dump"
+        },
+    ],
+    "loaders": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "ZIP",
+            "version": "1.0",
+            "handler": "load"
+        },
+    ],
+}
+
+def get_filename(path):
+    import os
+    return os.path.splitext(os.path.basename(path))[0]
+
+def load(file_object, annotations):
+    from pyunpack import Archive
+    import os
+    from tempfile import TemporaryDirectory
+
+    def convert_from_yolo(img_size, box):
+        def clamp(value, _min, _max):
+            return max(min(_max, value), _min)
+        xtl = clamp(img_size[0] * (box[0] - box[2] / 2), 0, img_size[0])
```

Comment:
Please add a comment about YOLO format to read/understand the code in the future